### ISSUE TITLE
Produce front-page that doesn't follow main template

### ISSUE THREAD

**Issue by IanMayo** [DeepBlueCLtd/Fi3ldMan#104]
I wish our document to have a front-page that doesn't contain any of the web-help elements - title, search, logo, headers/footers, etc.

We could have an `outputclass` in the `.dita` file page-level element, if that helps.

Note: there is some sample content to play with in `nav_trial\body-content` folder of `LegacyMan`. The `welcome.dita` is the page that I'd like to have no web-help decorations.

** Update:** we ***do*** still need the color-shaded header and footer bars.  It's just the web-help main title bar we don't need (with the home link, title, search box, links).

**Comment by frank-zimmermann** [comment]
In general, it could be an "empty" page, including the dita data, but no other components, right?

**Comment by IanMayo** [comment]
I _guess_ that if we had a page-level class element, we could hide the wh (webhelp) elements in CSS.

**Comment by frank-zimmermann** [comment]
Do you want to have a link to the "front-page" in the header in all other pages?

**Comment by IanMayo** [comment]
Hello @frank-zimmermann - no, I don't need a front page link on all pages.

**Comment by frank-zimmermann** [comment]
The attribute **outputclass** is a good hint. Will take that to suppress creating the link in the header and the main page.
I think I will do a modification in XSLT again.

**Comment by frank-zimmermann** [comment]
Hi @IanMayo,
the target folder is not included in the repo.
To suppress the topic-page in the header, we need to add the attribute **toc="no"**
When the format is "dita", the HTML for topic-page also have all CSS information, means a header and footer.

But we can change the **format** to "html". You then also need to create "**front-page.htm**" inside the **target/dita** folder.
With that, you can write directly HTML content into it.

**OR:**
Do you want to write dita content to it?

`    <?xml version="1.0" encoding="UTF-8"?>
    <!DOCTYPE map PUBLIC "-//OASIS//DTD DITA Map//EN" "map.dtd">
    <map>
        <title>Field Manual v4 Feb 2022</title>
        <topicmeta/>
        <topicref href="./front-page.htm" format="html" toc="no"/>
        <topicref href="./welcome.dita" format="dita">
            <topicmeta/>
        </topicref>
        <topicref href="./regions/QuickLinksData/Contents_Index.dita" format="dita">
            <topicmeta/>
        </topicref>
        <topicref href="./regions/regions.dita" format="dita">
            <topicmeta/>
        </topicref>
        <topicref href="./regions/Britain_Cmplx/unit_anchors.dita" format="dita">
            <topicmeta/>
        </topicref>
        <topicref href="./regions/Britain_Cmplx/Phase_F.dita" format="dita">
            <topicmeta/>
        </topicref>
        <topicref scope="external" href="https://deepbluecltd.github.io/LegacyMan/data/Introduction/ContentsPage.html" format="html">
            <topicmeta>
                <linktext>Legacy Content</linktext>
            </topicmeta>
        </topicref>
    </map>
`

![image](https://github.com/DeepBlueCLtd/Fi3ldMan/assets/49056485/1ec03228-b03c-4d5a-af2c-e44b95f3f52c)

**Comment by IanMayo** [comment]
Hello Frank, I did start off by creating a unique html page, but the DITA publish process doesn't pull the the images into the deployment, and they have to be addressed using a URL that I don't expect my authors to understand/recognise.

So, having this welcome page as DITA is my preferred outcome.

I'm going to update our mock content this afternoon to more closely match the current state of the target content.

**Comment by frank-zimmermann** [comment]
Then I will investigate some time to handle it as `.dita` by using the `outputclass`-attribute, as you suggested it.

**Comment by IanMayo** [comment]
Sorry Frank - when I got onsite I realised I was mistaken above.  It was having it in `.dita` that prevented the image dependencies from being picked up.  The _working_ version is for me to have a `welcome.html` that my python code parses and generates `welcome.dita`.   This, of course is then published to `welcome.html` in WebHelp format.

(I know the _round-trip_ may seem silly, but the original `welcome.html` contains hundreds of lines of DreamWeaver-generated cruft and unnecessary layout/GUI elements it's good to get rid of).

Anyway - the _crux_ of this task is to hide the web-help banner from `welcome.html`.  If we can do it via `fieldman.css` - then great :-D
